components/navBar/index.tsx

```
@@ -935,10 +806,10 @@
       setLocalId('');
 
       router.push('/authentication/login');
-    } catch (error) {}
+    } catch (error) { }
```

ESLint

[error] Delete `·`

(prettier/prettier)